### ISSUE TITLE
Run Inference

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/PVItaly#7]
## Detailed Description
Given a trained model run inference with the follow objectives
- Produce csv ready to hand over to client. 
- run metrics on test set, understand the MAE

## Context
Useufl to run on test set, so we are not running on training set

## Possible Implementation
CSV shouwl ahve the following columns
- ocf_id
- target_time_utc, with timezone to be clear. The time the forecast is for. 
- creation_time_utc, the time at which the forecast was made
- solar_generation_forecast_kw, the forecast value.
- solar_generation_actual_kw, the actual value.

**Comment by peterdudfield** [comment]
I think to make things simplier for the client it would be good to have

- ocf_id
- datetime_utc
- solar_generation_actual_kw
- solar_generation_forecast_kw_15m
- solar_generation_forecast_kw_30m
- solar_generation_forecast_kw_45m
- solar_generation_forecast_kw_60m
- ......
- solar_generation_forecast_kw_225m
- solar_generation_forecast_kw_240m

**Comment by peterdudfield** [comment]
Ill keep this open as I need to run inference for the final model

**Comment by peterdudfield** [comment]
@all-contributors please add @carderne  for code
